src/user-accounts/CreateNewAccount.js

```diff
@@ -19,36 +23,74 @@ export default class CreateNewAccount extends PureComponent {
 
   constructor(props) {
     super(props);
+    this.CRAccountSettings = undefined;
     this.state = {
       selected_currency: '',
       currency_error: false,
+      disable_create_account: false
     };
   }
 
   onCurrencyChange = (e: SyntheticEvent) => {
     this.setState({ [e.target.id]: e.target.value });
   }
 
+  addCRToCRAccount = async () => {
+    const s = this.CRAccountSettings;
+    // Fill in the minimum required info to add an account, cause backend
+    // ignores existing fields when creating from real-money account.
+    const createAccountParams = {
+      salutation: s.salutation,
+      date_of_birth: moment.unix(s.date_of_birth).format('YYYY-MM-DD'),
+      phone: s.phone,
+      residence: s.country_code,
+      last_name: s.last_name,
+      address_city: s.last_name,
+      address_line_1: s.address_line_1,
+      first_name: s.first_name,
+      currency: this.state.selected_currency
+    };
+    try {
+      const response = await api.createRealAccount(createAccountParams);
+      addNewAccount(response.new_account_real);
+      location.reload();
+    } catch (e) {
+      this.setState({ currency_error: true });
+      this.setState({ disable_create_account: false });
+    }
+  }
+
   onRedirectToAccountOpening = () => {
```

Comment:
@brucebinary Looks Good, thanks, 
I think with this behavior better to rename this function to submitCreateAccount or something that makes sense more, what do you think?

Reply:
@nazaninreihani I think `submitCreateAccount` is too general, since the function is specifically used to add a CR account only when there is an existing CR account. How about `submitCreateCRAccount`?

sry for the late reply.